Documentation/README.md

```diff
@@ -1,20 +1,99 @@
 # Resources
-
 See the following resources to get started.
 
-## .NET Core
+## System.Device.* APIs
 
-* [.NET Core documentation](https://docs.microsoft.com/dotnet/)
-* [Install .NET Core on Raspberry Pi](https://github.com/dotnet/core/blob/master/samples/RaspberryPiInstructions.md)
+* [Device Bindings](https://github.com/dotnet/iot/tree/master/src/devices) - Includes a collection of APIs representing a range of sensors, displays and human interface devices based on System.Device.* APIs.
```

Comment:
Should we also have another bullet point for the main library here? That way you sort of have the three sections of our repo here: Main library, device bindings, and the CLI tool (which in the future can just become a collection of tools)

Reply:
That is the plan.  If possible, I was going to start an overview doc (on basics of API and infrastructure details).  This is why I created https://github.com/dotnet/iot/issues/100.

I recommend going ahead and checking these changes as I'm sure there will be more helpful links to add in this markdown.  I'll keep a collection of those and add to a future PR when adding the overview doc.

Reply:
Agreed, we can add that other link later then. That said, I would like if you could address the other two comments I left if you have a chance.